docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -889,15 +890,44 @@ testcafe chrome my-tests --sf
 
 ### --ts-config-path \<path\>
 
-Enables TestCafe to use a custom [TypeScript configuration file](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options) and specifies its location.
+Deprecated as of [TestCafe v.X.X.X] in favour of the --compiler-options parameter.
+
+*Related configuration file property*: [tsConfigPath](configuration-file.md#tsconfigpath).
+
+### --compiler-options \<options\>
+
+Modifies TypeScript compilation settings.
+
+To list multiple parameters, separate them with semicolons. Enclose values that contain spaces in quotes.
 
 ```sh
-testcafe chrome my-tests --ts-config-path /Users/s.johnson/testcafe/tsconfig.json
+testcafe chrome my-tests --compiler-options typescript.options.lib=ES5,WebWorker;typescript.typesRoot='this value contains spaces'
 ```
 
-You can specify an absolute or relative path. Relative paths are resolved against the current directory (the directory from which you run TestCafe).
+The `--ts-config-path` option is now deprecated. Specify the `typescript.configPath` compiler option to load an external TypeScript configuration file:
```

Comment:
I guess the `The `--ts-config-path` option is now deprecated...` paragraph can be removed because it's description of the 'compiler-options' feature.